### ISSUE TITLE
Is it possible to run inference on a single .bin file without importing all the KITTI dataset?

### ISSUE THREAD

**Issue by chrispiro** [happinesslz/TANet#12]
Hello,

Congrats on your work! I would like to use the pre-trained model to perform inference on just few .bin files of the KITTI dataset (because I have limits of disk space).

Is this possible?

If I run creata_data.py I get missing '0000x.bin' error.

Do I need to modify something in particular in the .config files?

Thank you in advance!

**Comment by happinesslz** [comment]
@chrispiro  Default as using all  samples on validation set.  For the limitation of disk space for evaluating the model on validation set, you could modify the file names in [val.txt](https://github.com/happinesslz/TANet/blob/master/pointpillars_with_TANet/second/data/ImageSets/val.txt) . For example, if delete the file name in "val.txt" except "000001", it means that you only want to evaluate the "000001.bin" point cloud.

**Comment by chrispiro** [comment]
Thank you very much @happinesslz , it works!

I have another question, is there a pre-trained model for all the classes?

Or if I want to use the ped_cycle pre-trained model, should I use adam_optimizer-222720.tckpt? Or voxelnet-222720.tckpt?
And what .config files?

Thank you in advance!

**Comment by happinesslz** [comment]
@chrispiro  For ped_cycle pre-trained model in pointpillars_with_TANet, the corresponding config file is [xyres_16.proto](https://github.com/happinesslz/TANet/blob/master/pointpillars_with_TANet/second/configs/tanet/ped_cycle/xyres_16.proto). Note that you need specify the PATH of your dataset in the config file.

**Comment by chrispiro** [comment]
Ok thanks! @happinesslz
